app/src/main/kotlin/com/softteco/template/ui/feature/bluetooth/BluetoothViewModel.kt

```diff
@@ -52,6 +55,34 @@ class BluetoothViewModel @Inject constructor() : ViewModel() {
         }
     }
 
+    fun disconnectFromDevice() {
+        bluetoothHelper.disconnectFromDevice()
+    }
+
+    fun registerReceiver() {
```

Comment:
It would be good to add some documentation for functions, it's not clear why some methods need